### DISCUSSION TITLE
API gateway

### DISCUSSION THREAD

**Discussion by sytskevanhasselt** [Klantinteractie-Servicesysteem/KISS-frontend#34] [General]
We hebben een API gateway nodig om voor het leggen van connecties en translatie (het omzetten van berichten) zodat we vanuit de applicatie kunnen bouwen op standaard koppelvlakken (de gateway vertaalt deze vervolgens naar niet-standaard koppelvlakken of oude koppelvlakken). Voor de API Gateway zou bijvoorbeeld gekozen kunnen worden uit Layer7, Opentunnel, Mijn gemeente gateway, OpenZaakBrug en Conductor. Onze voorkeur gaat uit naar Conductor zoals ook in ons plan van aanpak is beschreven.

**Comment by rubenvdlinde** [comment]
Conduction geeft presentatie

- [ ] Waarom gateway? welke problemen lost je op
- [ ] Waarom conductor? vergelijking met andere gateways
- [ ] Veiligheid en afhankelijkheid
- [ ] Wat hoort er wel en niet bij de opdracht (afbakening adapters)
- [ ] Tekening op vijflagen model web gateway vs api gateway

Wie moeten er aan tafel?

- [ ] Martin
- [ ] Ruben
- [ ] Kees
- [ ] Martijn
- [ ] Hugo
- [ ] Frank

Deze pakken we komende week op
